magda-web-client/src/api-clients/RegistryApis.ts

```diff
@@ -27,15 +27,11 @@ export function fetchOrganization(publisherId: string): Promise<Publisher> {
 }
 
 export async function ensureAspectExists(id: string, jsonSchema: any) {
-    await request(
-        "PUT",
-        `${config.baseUrl}api/v0/registry-auth/aspects/${id}`,
-        {
-            id,
-            name: jsonSchema.title,
-            jsonSchema
-        }
-    );
+    await request("PUT", `${config.registryAuthApiUrl}aspects/${id}`, {
```

Comment:
For some reason this is going to `/registry` and not `/registry-auth`

Reply:
It seems this will be fixed by your commit in this PR:
https://github.com/magda-io/magda/pull/2411
I will probably merge that once it goes into the master to avoid creating more reference points of the legacy `/registry-auth`

Reply:
I think it's probably best to just fix this and let that one hit a merge conflict... it might be a while 'til that one's ready for merge because it's pretty huge and pretty complex whereas this one's quite small and simple.